packaging/debian/changelog

```diff
@@ -1,3 +1,9 @@
+mixxx (2.4.2-1~focal) focal; urgency=medium
+
+  * Build of 2.4.2
+
+ -- RJ Skerry-Ryan <rryan@mixxx.org>  Tue, 26 Nov 2024 23:29:32 +0000
+
```

Comment:
so this is actually missing in 2.4 branch?

Reply:
This is auto-generated with a current time stamp during the Launchpad upload, but not committed to git. We need to do it manually at any time before a new release, it in the change-log.

Reply:
Ah okay, thanks!
(I feel like I wondered about this for some previous release :laughing: hoepfully remember it this time)